scheduler/src/cook/mesos/util.clj

```diff
@@ -26,7 +26,8 @@
             [datomic.api :as d :refer (q)]
             [metatransaction.core :refer (db)]
             [metrics.timers :as timers]
-            [plumbing.core :as pc :refer (map-vals map-keys)])
+            [plumbing.core :as pc :refer (map-vals map-keys)]
+            [cook.mesos.pool :as pool])
```

Comment:
I hate to say this, but please sort this :)

Reply:
Done